src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +23,15 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @PutMapping
```

Comment:
I suggest to include `id` of the wallet to update in the endpoint. In that case we have to add second argument @Pathvariable Long id to the `updateWallet()` method and replace `@RequestBody @Valid UpdateWalletDTO updateWalletDto` with the object of existing `CreateWalletDTO.class` - In that case we can rename this class to `InputWalletDTO.class`